src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -471,13 +485,15 @@ This means you should very likely pin the platform ${chalk.greenBright(
 ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
     }
 
+    // TODO What and why?
     if (process.platform !== 'win32') {
       plusX(prismaPath)
     }
 
     return prismaPath
   }
 
+  // TODO
```

Comment:
Fixed as in "not broken" or fixed as in "written down"?
If any of these, why and how and where?

Reply:
```suggestion
  // TODO Fixed as in "not broken" or fixed as in "written down"? If any of these, why and how and where?
```